pyuploadcare/api/entities.py

```diff
@@ -1,11 +1,15 @@
+import re
 from datetime import datetime
 from decimal import Decimal
 from enum import Enum
 from typing import Dict, List, Optional, Tuple
 from uuid import UUID
 
+import pydantic
 from pydantic import BaseModel, EmailStr, PrivateAttr
```

Comment:
why just don't add `ConstrainedStr` to the second import?

Reply:
It would be the simplest way, totally!